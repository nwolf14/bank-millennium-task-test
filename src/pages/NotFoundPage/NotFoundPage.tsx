import { FC } from 'react';

// components
import Page from '../../components/Page/Page';

const NotFoundPage: FC = () => <Page>NotFoundPage</Page>;

export default NotFoundPage;
