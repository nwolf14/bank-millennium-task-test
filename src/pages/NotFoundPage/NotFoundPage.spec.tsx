import { render } from '@testing-library/react';

// components
import NotFoundPage from './NotFoundPage';

describe('NotFoundPage', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(<NotFoundPage />);

    expect(asFragment()).toMatchSnapshot();
  });
});
