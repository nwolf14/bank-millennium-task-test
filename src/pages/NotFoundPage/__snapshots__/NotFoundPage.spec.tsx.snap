// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage matches snapshot 1`] = `
<DocumentFragment>
  NotFoundPage
</DocumentFragment>
`;
